### PR TITLE
Feature 1 UI bootstrapping

### LINKED ISSUE #1: UI bootstrapping
As a developer I would like to have the UI bootstrapped

### PR DESCRIPTION
**Pull request by Chi-Chu319**
resolves #1

### CODE REVIEW

File: src/components/layouts/account-item.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  /**
+   *  Component render
+   */
+  return (
+    <>
+      <ListItemButton
+        onClick={ ({ currentTarget }) => setMenuAnchor(currentTarget) }
+      >
+        <ListItemText sx={{ color: theme.palette.primary.main }} primary={ strings.header.account }/>
```
Comment (lauriLukkarinen) [inline src/components/layouts/account-item.tsx]:
```suggestion
        <ListItemText
          sx={{ color: theme.palette.primary.main }}
          primary={ strings.header.account }
        />
```

File: src/components/layouts/header.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { AppBar, Typography, List } from "@mui/material";
+import strings from "localization/strings";
+import React from "react";
+import { LogoContainer, HeaderToolbar } from "styled/layouts/header";
+import PallasmarjaLogoPath from "../../resources/svg/pakkasmarja-logo.svg";
```
Comment (lauriLukkarinen) [inline src/components/layouts/header.tsx]: Fix typo

File: src/styled/layouts/navigation-item.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ */
+export const NavigationButton = styled(ListItemButton, {
+  label: "navigation-button"
+})(({ theme }) => ({
+  padding: theme.spacing(2),
+  [theme.breakpoints.up("md")]: {
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
```
Comment (lauriLukkarinen) [inline src/styled/layouts/navigation-item.tsx]:
```suggestion
    padding: theme.spacing(1, 2),
```

**Review by lauriLukkarinen — CHANGES_REQUESTED** [review]

File: src/components/layouts/header.tsx (hunk 2 above)
Comment (Chi-Chu319) [inline src/components/layouts/header.tsx]:
```suggestion
import PakkasmarjaLogoPath from "../../resources/svg/pakkasmarja-logo.svg";
```

File: src/components/layouts/header.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ */
+const Header: React.FC = () => (
+  <AppBar
+    elevation={ 0 }
+    position="static"
+    sx={{ zIndex: theme => theme.zIndex.drawer + 1 }}
+  >
+    <HeaderToolbar>
+      <LogoContainer>
+        <img alt="Pakkasmarja logo" src={ PallasmarjaLogoPath }/>
```
Comment (Chi-Chu319) [inline src/components/layouts/header.tsx]:
```suggestion
        <img alt="Pakkasmarja logo" src={ PakkasmarjaLogoPath }/>
```

**Review by lauriLukkarinen — APPROVED** [review]

**Pull request merged**